barretenberg/cpp/src/barretenberg/vm2/common/aztec_types.hpp

```diff
@@ -31,6 +31,26 @@ enum TransactionPhase {
 
 using InternalCallId = uint32_t;
 
+/**
+ * Enum for environment variables, representing the various environment values
+ * that can be accessed by the AVM GETENVVAR opcode.
+ */
+enum class EnvironmentVariable {
+    ADDRESS = 0,
```

Comment:
Do we need to specify the numbers for some reason? Otherwise C++ does it for you and it's the standard.